tasks/fine_tuning_sbert/notebooks/Fine_tuning_SBERT_with_callback.ipynb

```diff
@@ -1,3160 +1,2839 @@
+            "text/plain": [
+              "<Figure size 432x288 with 1 Axes>"
+            ]
+          },
+          "metadata": {
+            "tags": [],
+            "needs_background": "light"
+          }
+        },
+        {
+          "output_type": "stream",
+          "text": [
+            "Label counts:\n",
+            "{'Incentive': 0.49146757679180886, 'not_Incentive': 0.5085324232081911}\n"
+          ],
+          "name": "stdout"
+        }
       ]
-     },
-     "metadata": {
-      "tags": []
-     },
-     "output_type": "display_data"
     },
     {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "============================================================\n",
-      "Current Score is: 0.9382716049382716\n",
-      "Current ACC_LIST is: [0.8148148148148148, 0.9444444444444444, 0.9382716049382716]\n",
-      "Epoch: 2\n",
-      "Best score: 0.9444444444444444\n",
-      "============================================================\n"
-     ]
+      "cell_type": "code",
+      "metadata": {
+        "id": "v1MTStwySrYL"
+      },
+      "source": [
+        "# PASTE YOUR WEIGHTS & BIASES KEY HERE\n",
```

Comment:
Let's add a comment saying that we should not forget to delete them, as I did :D

Reply:
Added the comments for this 😉